esileapclient/tests/functional/utils/dummy_node.py

```diff
@@ -0,0 +1,37 @@
+import os
+import tempfile
+
+
+class DummyNode():
+    def __init__(self, node_dir, project_uuid=''):
+        if not os.path.exists(node_dir):
+            os.mkdir(node_dir)
+        elif not os.path.isdir(node_dir):
+            raise NotADirectoryError('Invalid value for dummy_node_dir: %s'
+                                     % node_dir)
+
+        dummy_node_info = [
+            '{',
+            '   "project_owner_id": "%s",' % project_uuid,
+            '   "server_config": {',
+            '       "example_attribute": "example server config",',
+            '       "cpu_type": "Intel Xeon",',
+            '       "cores": 16,',
+            '       "ram_gb": 512,',
+            '       "storage_type": "Samsung SSD",',
+            '       "storage_size_gb": 1024',
+            '   }',
+            '}']
```

Comment:
If you use json.dump, you can specify dummy_node_info as a dict and then simply dump that dict into the file. It'll eliminate most of the next six lines as well.

Reply:
oh neat. that sounds a lot cleaner!